examples/3d/modelNoiseExample/src/ofApp.cpp

```diff
@@ -105,7 +105,12 @@ void ofApp::drawWithMesh(){
 	float liquidness = 5;
 	float amplitude = mouseY/100.0;
 	float speedDampen = 5;
-	vector<ofVec3f>& verts = mesh.getVertices();
+    auto t_vect = mesh.getVertices();
```

Comment:
I fear this won't fix the issue you've found. Note that `verts` was a _reference_ to a vector of vertices, i.e. a shorthand to directly manipulating the vertices inside the mesh. 

Could you try and check if the following line fixes the compile issues:

`auto & verts = mesh.getVertices();`

This wouldn't need the "manual" conversion.